trl/trainer/rloo_trainer.py

```diff
@@ -300,7 +305,6 @@ def repeat_generator():
                 queries = data["input_ids"].to(device)
                 queries = queries.repeat(args.rloo_k, 1)
                 context_length = queries.shape[1]
-                query_responses = []
```

Comment:
you removed this because it's not used?

Reply:
Yes, its immediately overwritten https://github.com/huggingface/trl/blob/21d5baf338be52e21af95d8d0c6cbc4968238181/trl/trainer/rloo_trainer.py#L311